### PR TITLE
test/release/examples/benchmarks/shootout/README: two tweaks

### PR DESCRIPTION
**Pull request by cassella**
List regexdna with the .chpl extension like all the others.

Reference start_test instead of sub_test.

### CODE REVIEW

**Comment by cassella** [comment]
I assume that sentence meant to mention start_test.  I see sub_test scripts, but not in $CHPL_HOME/util, and not looking like meaningful places to point someone to get started with testing.

File: test/release/examples/benchmarks/shootout/README [hunk 1]
```diff
@@ -31,7 +31,7 @@ repository under the test/studies/shootout/ directory.
 
 The provided Makefile can be used to compile the programs, or they can
 be run in correctness or performance modes using the Chapel testing
-system (see $CHPL_HOME/util/sub_test).
+system (see $CHPL_HOME/util/start_test).
```
Comment (lydia-duncan) [inline test/release/examples/benchmarks/shootout/README]: While you're right that the original intention was to point the reader at start_test, start_test no longer contains the instructions for how to run a particular job.  With that in mind, it's probably more useful to point the reader at $CHPL_HOME/doc/developer/bestPractices/TestSystem.rst (being sure to note that this file is only available in the github repository and not the release tarball)

**Comment by cassella** [comment]
I took a shot at that.

I think I lack the context wrt the shootout and the audience for
the README submitted to it to know if this is a suitable way to
refer to it though.

**Comment by lydia-duncan** [comment]
That looks fine to me, thanks!

**Pull request merged**